src/face_detection_module.py

```diff
@@ -138,4 +138,9 @@ async def do_command(self,
                         *,
                         timeout: Optional[float] = None,
                         **kwargs):
-        raise NotImplementedError
\ No newline at end of file
+        if command["command"] == "recompute_embeddings" :
+            self.identifier.known_embeddings = dict()
+            self.identifier.compute_known_embeddings()
+            LOGGER.error("Recompute Embeddings!")
```

Comment:
Being nosy here but do we want this to be a LOGGER.info() instead?

Reply:
The difference being it doesn't show as if you messed something up

Reply:
you are absolutely right :-)